templates/admin/dashboard/network.blade.php

```diff
@@ -26,6 +25,7 @@ class="button button-primary"
 						</a>
 					@endif
 				</div>
+				<div class="wp-header-end"></div>
```

Comment:
WordPress uses the .wp-header-end class as a reference point for placing admin notices. See https://github.com/WordPress/WordPress/blob/4959927285104dd4b032ffabd9fe5f58a0fe9db7/wp-admin/js/common.js#L1081-L1091